library/download-manager/src/main/java/org/cru/godtools/download/manager/DownloadProgressLiveData.kt

```diff
@@ -3,7 +3,7 @@ package org.cru.godtools.download.manager
 import androidx.annotation.MainThread
 import androidx.lifecycle.MutableLiveData
 
-class DownloadProgressLiveData : MutableLiveData<DownloadProgress?>() {
+internal class DownloadProgressLiveData : MutableLiveData<DownloadProgress?>() {
```

Comment:
Similar question to my others, what does ```internal``` add to this?

Reply:
`internal` is a Kotlin visibility modifier. in this case it is restricting visibility of the `DownloadProgressLiveData` class to be visible within the `download-manager` module only.

the class it extends is still externally visible, but making this `internal` makes sure we only use this directly within the `download-manager`